### DISCUSSION TITLE
Integrate with Flodesk email marketing platform

### DISCUSSION THREAD

**Discussion by MagnoliaRabbit** [elementor/elementor#28376] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I use Flodesk as my email marketing platform and I'm stuck paying another $30/mo for Zapier so I can run automations from the opt-in forms on my website... would love it if it could integrate directly so I can stop using Webhooks. 

### Describe the solution you'd like.

I'd like for Flodesk to integrate the same way ConvertKit, MailerLite, ActiveCampaign, etc. integrate so that Elementor automatically connects and populates groups, tags, etc. for your form actions. 

### Describe alternatives you've considered.

I'm using Zapier to work around it right now.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by Webtica** [comment]
Hi @MagnoliaRabbit ,

We made a integration for Brevo: https://wordpress.org/plugins/integration-for-elementor-forms-sendinblue/
If you want we can make the same integration for Flodesk.
